pkg/kubelet/volume_host.go

```diff
@@ -169,24 +170,31 @@ func (kvh *kubeletVolumeHost) CSIDriverLister() storagelisters.CSIDriverLister {
 	return kvh.csiDriverLister
 }
 
-func (kvh *kubeletVolumeHost) CSIDriversSynced() cache.InformerSynced {
-	return kvh.csiDriversSynced
+func (kvh *kubeletVolumeHost) CSIDriversSynced() (bool, error) {
+	if kvh.csiDriversSynced == nil {
+		return false, errors.New("csiDriversSynced not found on kubeletVolumeHost")
+	}
+	if synced := kvh.csiDriversSynced(); !synced {
+		return false, nil
+	}
+	return true, nil
```

Comment:
is it possible to simplify L177-180:
```
return kvh.csiDriversSynced(), nil
```

Reply:
Thanks for your suggestion, I have modified the code